### ISSUE TITLE
Adyens monolog version crashes with standard vendors monolog version

### ISSUE THREAD

**Issue by ifbidev** [Adyen/adyen-shopware6#398]
**Describe the bug**
We make use of the monolog version provided by symfony that is provided with  standard shopware installation ( currently version 6.5.3.1).
After installing the adyen plugin (version 3.12.1) our shop crashes with 500 Error

**Versions**
Shopware version:  6.5.3.1
Plugin version:  3.12.1

monolog version in Adyen Plugin:  2.9.1

monolog version in shopware vendor : 3.3.1

**Comment by alexbaat** [comment]
Since 6.5 you only need to require the dependencies in the composer.json and not add a vendors folder. If you follow this https://github.com/shopware/platform/blob/6.5.0.0/UPGRADE-6.5.md#new-executecomposercommands-option-for-plugins the dependencies will be installed in the root vendor folder and you will not have duplicate monolog.

**Comment by ifbidev** [comment]
Yes but the  adyen plugin zip brings its own dependencies as described above that collide with the shop's vendor directory.

**Comment by alexbaat** [comment]
Yes, I know. My description is meant for Adyen on how to solve it in next version.

**Comment by ifbidev** [comment]
Due to an incredibly disappointing decision by Adyen to deny us onboarding, 
I am closing this issue. 
It is now irrelevant to us, and we can't help but feel profoundly let down by the outcome.
